frontend/src/poc-code/testK8s/TestK8s.tsx

```diff
@@ -0,0 +1,209 @@
+            alert('Could not find namespace; you are likely not able to do much as we are targeting "default"');
+          }
+          return data;
+        });
+        break;
+      case ActionType.CREATE:
+        promise = k8sCreateResource({
+          model: ConfigMapModel,
+          data: testConfigMapData,
+        });
+        break;
+      case ActionType.GET:
+        promise = k8sGetResource({
+          model: ConfigMapModel,
+          name: testConfigMapMetadata.metadata.name,
+          ns: testConfigMapMetadata.metadata.namespace,
+        });
+        break;
+      case ActionType.PATCH:
+        promise = k8sPatchResource({
+          model: ConfigMapModel,
+          resource: testConfigMapMetadata,
+          data: [
+            {
+              op: 'replace',
+              path: '/data/test',
+              value: 'false',
+            },
+          ],
+        });
+        break;
+      case ActionType.PUT:
+        promise = k8sUpdateResource({
+          model: ConfigMapModel,
+          name: testConfigMapMetadata.metadata.name,
+          ns: testConfigMapMetadata.metadata.namespace,
+          data: { ...testConfigMapData, data: { ...testConfigMapData.data, new: 'prop' } },
+        });
+        break;
+      case ActionType.DELETE:
+        promise = k8sDeleteResource({
+          model: ConfigMapModel,
+          resource: testConfigMapMetadata,
+        });
+        break;
+      case null:
+        // ignore effect
+        break;
+      default:
+        // this shouldn't happen, catch state for missed cases
+        throw new Error('uh oh!');
+    }
+    promise
+      ?.then((data) => {
+        setStatus(`${action} response:`);
+        setR(data);
+        console.debug(`++++${action}!`, data);
+      })
+      .catch((err) => {
+        console.error(`++++failed ${action}`, err);
+        setStatus('failed call');
+        setR(null);
+      })
+      .finally(() => {
+        setAction(null); // prevent the hook for re-firing on name change
+      });
+  }, [action, name, namespace]);
+
+  return (
+    <PageSection>
+      <TextInput placeholder="ConfigMap name" onChange={(v) => setName(v)} value={name} />
+      <div>
+        <p>Test calls -- predefined data; use the above input to make/update/get multiple ConfigMaps</p>
+        {Object.values(ActionType).map((v) => (
+          <React.Fragment key={v}>
+            <Button isDisabled={v !== ActionType.LIST && namespace === 'default'} onClick={() => setAction(v)}>
+              {v}
+            </Button>{' '}
+          </React.Fragment>
+        ))}
+        In `{namespace}` namespace
+      </div>
+      {/*{hookResource ? (*/}
+      {/*  <>*/}
+      {/*    {Array.isArray(hookResource) && <p>{hookResource.length} item(s) in the array</p>}*/}
+      {/*    <pre>{JSON.stringify(hookResource, null, 2)}</pre>*/}
+      {/*  </>*/}
+      {/*) : (*/}
+      {/*  'No watched resource'*/}
+      {/*)}*/}
+      {r && (
+        <>
```

Comment:
No need to mix empty elements and `React.Fragment` would be better to use `React.Fragment`